models/duration.js

```diff
@@ -0,0 +1,18 @@
+// data for duration 
+
+const mongoose = require('mongoose');
+
+const Schema = mongoose.Schema;
+
+const durSchema = new mongoose.Schema({
+    duration: {type: Number, min: 1, max: 60, required: true}
+});
+
+const findDuration = module.exports = mongoose.model('duration', durSchema);
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/Nlopez4/fitness_tracking_app/pullRequest?prid=7827980)
```suggestion
const findDuration = module.exports = mongoose.model("duration", durSchema);
```